temoa/model_checking/unit_checking/screener.py

```
@@ -0,0 +1,270 @@
+    Run series of units screens on the database
+    :param db_paths: the abs path(S) to the database(s)
+    :param report_path: abs path to write the report to.  If None, no report is written
+    :return: indicator of whether all checks passed "cleanly" or not
+    """
+    all_clear = True
+    report_entries = []
+    table_units = {}
+    """Table name : {tech | commodity: units}"""
+    for db_path in db_paths:
+        if not db_path.is_file():
+            raise FileNotFoundError(f'Database file not found: {db_path}')
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)